### ISSUE TITLE
Arabic Font

### ISSUE THREAD

**Issue by malekairmaroc7** [jenskutilek/free-fonts#3]
Does a (noto/droid sans/...) font exist like this:
https://imgur.com/a/xFu1ArL

A noto sans/droid sans arabic font that don't have the Roboto style. I searched the whole web but no success

**Comment by jenskutilek** [comment]
Not sure what you mean exactly ... the image seems to show the Droid Kufi Arabic style. It does not contain any Latin glyphs, so maybe you see Roboto as a fallback font?

**Comment by malekairmaroc7** [comment]
Yes. It displays roboto fallback but i want the droid sans styled latin. I can't find it on the internet
